build/base/Dockerfile

```diff
@@ -0,0 +1,102 @@
+ARG BASEOS
+ARG DOCKERBASEREGISTRY
+FROM ${DOCKERBASEREGISTRY}${BASEOS}
+
+# For RHEL8 all arguments used in main code has to be specified after FROM
+ARG BASEOS
+ARG RELVER
+ARG DFSET
+ARG PACKAGER
+
+MAINTAINER Crunchy Data <info@crunchydata.com>
+
+LABEL vendor="Crunchy Data" \
+	url="https://crunchydata.com" \
+	release="${RELVER}" \
+	org.opencontainers.image.vendor="Crunchy Data" \
+	os.version="7.7"
```

Comment:
This label doesn't seem relevant anymore.

Reply:
That is possible, but that is not something I was going to change from what it is in the original Dockerfile.  We can have another PR for stripping those out if we need to once we verify that the scanners aren't looking for it.

Reply:
I think better would be to add it to a build var so we can toggle it. OK-ish if it's a separate commit.